components/Grid/grid.css

```diff
@@ -0,0 +1,37 @@
+ 
+  .grid-item {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 14rem;
+    height: 4.2rem;
+    background-color: var(--hold-color);
+    color: var(--color-white);
+
+}
+  
+
+  .content-grid-two {
```

Comment:
You can name .content-grid-two as .grid-two-cols and move it above .grid-items ruleset.
Similarly, for .content-grid-three as .grid-three-cols

Reply:
okay